pombola/south_africa/templates/south_africa/mp_attendance.html

```diff
@@ -0,0 +1,112 @@
+{% extends 'base.html' %}
+{% load compressed static %}
+
+{% block title %}MP Attendance{% endblock %}
+
+{% block extra_js_to_load %}
+    addExtraJS([ {% compressed_js 'attendance-table' %} null ]);
+{% endblock %}
+
+{% block css_headers %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static 'css/libs/datatables.min.css' %}" />
+{% endblock %}
+
+{% block body_attributes %} class="mp-attendance" {% endblock %}
+
+{% block content %}
+<h1 class="page-title">MP Attendance</h1>
+<div class="major-column">
+  <p>Explore the commitee meeting attendance records for the members of parliament.</p>
+</div>
+
+<div class='mp-attendance-filter'>
+  <div class="col-50">
+    <form class='select-year' action="{% url 'mp-attendance' %}" method="get">
+      {% csrf_token %}
+      <div class="filter-option">
+        <label>Select a year:</label>
+        <select name="year">
+         {% for yr in years %}
+          <option value="{{ yr }}" {% if yr == year %}selected{% endif %}>{{ yr }}</option>
+         {% endfor %}
+        </select>
+      </div>
+      <div class="filter-option">
+        <label>Choose a party:</label>
+        <select name="party">
+          <option value="" selected>All</option>
+         {% for pty in parties %}
+          <option value="{{ pty }}" {% if pty == party %}selected{% endif %}>{{ pty }}</option>
+         {% endfor %}
+        </select>
+      </div>
+      <input class="button" type="submit" value="Show attendance">
+    </form>
+  </div>
+
+  {% if party %}
+   <div class="col-50">
+    <h2>{{ aggregate_attendance }}%</h2>
+    <p>Average attendance for {{ party }}</p>
+   </div>
+  {% endif %}
+
+  <div class="clear"></div>
+
+  <hr>
+    <a href="{{ download_url }}">
```

Comment:
I think this could be simplified to:

    <a class="button download" href="{{ download_url }}">Download raw data</a>